config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -361,3 +361,45 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=40m
       - --use-logexporter
+
+# cluster volume load scaling by number of nodes
+- name: ci-kubernetes-storage-scalability-cluster-load-scale-by-node
+  tags:
+    - "perfDashPrefix: storage-cluster-load-scale-persistent-volume-by-node"
+    - "perfDashJobType: storage"
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-node: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: cluster-load-scale-persistent-volume-by-node
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190726-d319b3f-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --repo=k8s.io/perf-tests=master
+          - --root=/go/src
+          - --timeout=60
+          - --scenario=kubernetes_e2e
+          - --
+          - --check-leaked-resources
+          - --cluster=
+          - --extract=ci/latest
+          - --gcp-node-image=gci
+          - --gcp-nodes=1
+          - --provider=gce
+          - --test=false
+          - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+          - --test-cmd-args=cluster-loader2
+          - --test-cmd-args=--nodes=500
```

Comment:
I'm not familiar with the test-infra concepts, but what's the thing in here to provide you 500 nodes on 1-node gcp?

Reply:
Sorry, thats a mistake